candump.c

```diff
@@ -98,9 +99,15 @@
 const char col_on [MAXCOL][19] = {BLUE, RED, GREEN, BOLD, MAGENTA, CYAN};
 const char col_off [] = ATTRESET;
 
-static char *cmdlinename[MAXSOCK];
-static __u32 dropcnt[MAXSOCK];
-static __u32 last_dropcnt[MAXSOCK];
+struct if_info /* bundled information per open socket */
+{
```

Comment:
please move the opening bracket into the same line as the `if_info`

Reply:
Done, sorry for not implying coding style.